src/code/z_parameter.c

```diff
@@ -600,15 +604,15 @@ u8 Item_Give(PlayState* play, u8 item) {
         return item;
 
     } else if (item == ITEM_MAGIC_SMALL) {
-        Parameter_AddMagic(play, 0x18);
+        Magic_Add(play, MAGIC_NORMAL_METER / 2);
```

Comment:
I wouldn't necessarily say that the magic recovered from a magic jar is inherently related to the size of the meter.

Reply:
It's a little subjective, you could design magic to restore a fixed amount or a fraction of the total. At least with this interpretation, it saves an extra define and gives scale to how much magic is restored

Reply:
While I do agree it's not inherently tied to the size of the magic meter, I'd say it's clear that chose that value because it's half. I also think we should keep it this way because it gives someone reading the code context on how much magic "0"18" is restoring, in other words documentation. :)